cometsdk.go

```diff
@@ -20,6 +20,8 @@ const APPLICATION_VERSION string = "24.3.8"
 const APPLICATION_VERSION_MAJOR int = 24
 const APPLICATION_VERSION_MINOR int = 3
 const APPLICATION_VERSION_REVISION int = 8
+const USER_AGENT_HEADER string = "UserAgent"
+const DEFAULT_USER_AGENT string = "CometBackup/" + APPLICATION_VERSION
```

Comment:
The Comet Backup app itself uses this string as its user-agent already. I think it would be valuable to distinguish the User-Agent between the SDK and the app itself -

Our PHP SDK [sets the User-Agent header](https://github.com/CometBackup/comet-php-sdk/blob/master/Comet/Server.php) to `comet-php-sdk/1.x`, so for consistency I propose `comet-go-sdk/x.y.z` here. You can of course still override it when constructing the client.

Reply:
Yes, that's perfect, I'll update it.